librz/bin/p/bin_z64.c

```diff
@@ -64,10 +68,9 @@ typedef struct {
 	// BOOT CODE?
 } N64Header;
 
-static N64Header n64_header;
-
 static ut64 baddr(RzBinFile *bf) {
-	return (ut64)rz_read_be32(&n64_header.BootAddress);
+	N64Header *hdr = bf->o->bin_obj;
+	return (ut64)rz_read_be32(&(hdr->BootAddress));
```

Comment:
if the header is read correctly, then this is not needed and instead it should just return this:

```c
N64Header *hdr = bf->o->bin_obj;
return hdr->BootAddress;
```